src/threadpool-pthreads.c

```diff
@@ -438,9 +448,16 @@ struct pthreadpool* pthreadpool_create(size_t threads_count) {
 	pthread_once(&nacl_init_guard, nacl_init);
 #endif
 
+#if defined(_SC_NPROCESSORS_ONLN)
```

Comment:
If `threads_count == 0` and neither `_WIN32` nor `_SC_NPROCESSORS_ONLN` is defined, the thread pool will be left in an invalid state. Compilation error is preferable in this case.